src/CMakeLists.txt

```diff
@@ -46,7 +46,8 @@ endif()
 #
 option(NUPIC_IWYU "Enable include-what-you-use
   (http://include-what-you-use.org/). This requires the iwyu binary to be
-  discoverable by CMake's find_program.")
+  discoverable by CMake's find_program.  The easiest way to install iwyu on
+  OS X is to use https://github.com/jasonmp85/homebrew-iwyu")
```

Comment:
I think the OS X install comment isn't needed here since we have it in the README. But we should mention that CMake 3.3 is required.